Marlin/configuration_store.cpp

```diff
@@ -555,6 +555,7 @@ void Config_ResetDefault() {
 
   #if ENABLED(AUTO_BED_LEVELING_FEATURE)
     zprobe_zoffset = Z_PROBE_OFFSET_FROM_EXTRUDER;
+    update_software_endstops(Z_AXIS);
   #endif
```

Comment:
That seems to be problematic. Seems to be the same as in M851. We probably should not apply this immediately.  We don't move the nozzle nor shift a coordinate system. So the correction for this should apply during the next homing and/or probing.